frameworks/OCaml/opium/opium-haproxy.dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ocaml/opam2:alpine
+
+USER root
+
+RUN apk add --no-cache make m4 postgresql-dev libev-dev libffi-dev linux-headers
+
+WORKDIR /web
+
+RUN	opam init --disable-sandboxing --auto-setup --compiler 4.10.1\
```

Comment:
something didn't work with 4.11.1, but can't remember what (I had to make a ton of changes to finally make this build);
I'll double check 4.11.1 just in case